lib/WebService/MyAffiliates.pm

```diff
@@ -165,6 +165,15 @@ sub get_affiliate_id_from_token {
     return $token_info->{TOKEN}->{USER_ID};
 }
 
+sub get_affiliate_email_from_token {
```

Comment:
Please add test for this subroutine

Reply:
Also let us add POD for newly created subs.
https://wikijs.deriv.cloud/en/Backend/guides/perl/perl-style-guide#subroutine-comments

Reply:
Ah I put reviews on cpan :( , please address regentmarkets/cpan#2785 as well.